src/uwtools/drivers/support.py

```diff
@@ -15,8 +19,10 @@ def graph() -> str:
 def tasks(driver_class: type[Driver]) -> Dict[str, str]:
     """
     Returns a mapping from task names to their one-line descriptions.
+
+    :param driver_class: Class of driver object to instantiate.
     """
     return {
-        task: getattr(driver_class, task).__doc__.strip().split("\n")[0]
+        task: (getattr(driver_class, task).__doc__ or "").strip().split("\n")[0]
```

Comment:
Fixes a bug whereby an exception was raised if a task function/method did not provide a docstring.